sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -1064,6 +1063,68 @@ public String getEncryptionScope() {
             });
     }
 
+//    /**
```

Comment:
Feature is cut - need to comment out instead of mark as package private since the code references BlobContainerRenameOptions and since we need to park that as private this code becomes uncompilable